config/manager/deployment.yaml

```diff
@@ -59,6 +59,11 @@ spec:
         volumeMounts:
```

Comment:
I think we'll also need to come up with a better default value for the memory request, considering we now run an embedded database as well. Not sure what the actual footprint would be.